hack/generate-release-yaml.sh

```diff
@@ -20,6 +22,6 @@ for FILE in ${FILES[@]}
 do
 
 	echo -e "\n---\n" >> release.yaml
-	cat deploy/${FILE} >> release.yaml
+	cat ${FILE} >> release.yaml
```

Comment:
yeah this most likely will go away once we intergrate with the CSO, but better to keep it sensible in the interim